### PR TITLE
Add Ello

### PR DESCRIPTION
**Pull request by anotherus3r**
Added Ello.co website

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -40,6 +40,13 @@ websites:
       twitter: deviantart
       img: deviantart.png
       tfa: No
+
```
Comment (Jawshy) [inline _data/social.yml]: Please remove the spaces on this line.

**Review by Jawshy — CHANGES_REQUESTED** [review]
[Travis CI job log](https://travis-ci.org/2factorauth/twofactorauth/jobs/194074609): 

```
1. img/social/ello.png is not 32x32 pixels.
```

@randomeuropean, please reduce the size of the image to exactly 32 by 32 pixels.

**Comment by Carlgo11** [comment]
Please add their email as well. `email_address: suggestions@ello.co`

**Review by Jawshy — APPROVED** [review]
Thanks, @randomeuropean.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**